test/integration/user-management-api/applications_test.rb

```diff
@@ -32,14 +32,39 @@ def setup
     User.any_instance.stubs(:has_access_to_all_services?).returns(false)
     user  = FactoryBot.create(:member, account: @provider, admin_sections: ['partners'])
     token = FactoryBot.create(:access_token, owner: user, scopes: 'account_management')
+    service_2 = FactoryBot.create(:service, account: @provider)
+    service_3 = FactoryBot.create(:service, account: @provider)
+    application_plan_2 = FactoryBot.create(:application_plan, issuer: service_2)
+    application_plan_3 = FactoryBot.create(:application_plan, issuer: service_3)
+    application_2 = FactoryBot.create(:cinstance, plan: application_plan_2, user_account: @buyer)
+    FactoryBot.create(:cinstance, plan: application_plan_3, user_account: @buyer)
 
     get(admin_api_applications_path)
     assert_response :forbidden
     get admin_api_applications_path, params: { access_token: token.value }
     assert_response :success
+    assert_select "applications/application", false
+
     User.any_instance.expects(:member_permission_service_ids).returns([@service.id]).at_least_once
+    get admin_api_applications_path, params: { access_token: token.value, service_id: @service.id + 1 }
```

Comment:
why `@service.id + 1` ? :thinking: 

what is this testing?

Reply:
just a service ID that doesn't exist or user has no access to. Requesting a single such service.

Reply:
Maybe we should try with both options? Or this is fine for you?

Reply:
Actually this should be `service_2.id`, I don't see how using a random other ID make sense.